src/views/v3/Bridge/Routes/RoutesDesktop.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useMemo } from 'react';
+import { useTheme } from '@mui/material';
+import Box from '@mui/material/Box';
+import IconButton from '@mui/material/IconButton';
+import Modal from '@mui/material/Modal';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import CloseIcon from '@mui/icons-material/Close';
+import { routes } from '@wormhole-foundation/sdk';
+
+import Button from 'components/v3/Button';
+import SingleRoute from 'views/v3/Bridge/Routes/SingleRoute';
+
+interface RoutesDesktopProps {
+  open: boolean;
+  onClose: () => void;
+  routesWithQuotes: string[];
+  highlightedRoute?: string;
+  quotes: Record<string, routes.QuoteResult<routes.Options> | undefined>;
+  fastestRoute: { name: string; eta: number };
+  cheapestRoute: { name: string; amountOut: bigint };
+  onRouteSelect: (route: string) => void;
+  onGasChange: (value: number) => void;
+  onSelectRoute: () => void;
+  selectButtonDisabled: boolean;
+}
+
+function RoutesDesktop({
+  open,
+  onClose,
+  routesWithQuotes,
+  highlightedRoute,
+  quotes,
+  fastestRoute,
+  cheapestRoute,
+  onRouteSelect,
+  onGasChange,
+  onSelectRoute,
+  selectButtonDisabled,
+}: RoutesDesktopProps) {
+  const theme = useTheme();
+
+  // Header section - static content, no need to memoize
+  const routesHeader = (
```

Comment:
Can be its own component `<SelectRouteHeader>` similarly `<SelectRouteButton>` below.